recipes/idds-common/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "idds-common" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/idds-common-{{ version }}.tar.gz
+  sha256: d9d205ca74453a0398491e096f885ae50294f326818d58ef9bc101e6e8db8aca
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - flake8
+    - nose
+    - pep8
+    - pip
+    - pytest
```

Comment:
These all look like test deps and pip is not usually needed at run time. Try removing those and unittest2.